support-services/src/main/scala/com/gu/support/redemption/DynamoTableAsync.scala

```diff
@@ -19,8 +19,9 @@ object DynamoTableAsync {
   val ProfileName = "membership"
 
   lazy val CredentialsProvider =  AwsCredentialsProviderChain.builder.credentialsProviders(
-     ProfileCredentialsProvider.builder.profileName(ProfileName).build,
-     InstanceProfileCredentialsProvider.builder.asyncCredentialUpdateEnabled(false).build
+    ProfileCredentialsProvider.builder.profileName(ProfileName).build,
+    InstanceProfileCredentialsProvider.builder.asyncCredentialUpdateEnabled(false).build,
+    EnvironmentVariableCredentialsProvider.create()
```

Comment:
this was needed when running in AWS, the lambda gets the credentials from the environment, but somehow AWS SDK v1 was picking them up regardless.